app/controllers/application_controller.rb

```diff
@@ -36,5 +50,10 @@ def authenticate_admin!
 
   def authorize
     redirect_to root_path, alert: 'Access denied' unless can? :manage, @question
+  end 
+
+  def reset_session
+      @_request.reset_session
   end
+
```

Comment:
yeah, forgot about this. was trying something with routes before deleting it all. will del.